featuretools/tests/primitive_tests/test_transform_primitive.py

```diff
@@ -711,10 +713,14 @@ def test_email_address_to_domain_empty_string():
 
 def test_email_address_to_domain_empty_series():
     email_address_to_domain = EmailAddressToDomain()
-    array = pd.Series([])
+    array = pd.Series([], dtype="category")
```

Comment:
Not sure exactly why these changes are included, but the input to this primitive function will be an `EmailAddress` and shouldn't a column with `EmailAddress` logical type have a dtype of `string`?